src/main/resources/test_files/runtime-spark/RumbleML/MLTransformerVectorAssembler2.jq

```diff
@@ -0,0 +1,8 @@
+(:JIQS: ShouldRun; Output="({ "id" : 1, "age" : 20, "weight" : 68.8, "features" : [ 20, 68.8 ] }, { "id" : 2, "age" : 35, "weight" : 72.4, "features" : [ 35, 72.4 ] }, { "id" : 3, "age" : 50, "weight" : 76.3, "features" : [ 50, 76.3 ] })" :)
+let $transformer := get-transformer("VectorAssembler")
+let $rdd-data := json-file("./src/main/resources/queries/rumbleML/sample-ml-age-weight-data.json")
+for $i in $transformer(
+  $rdd-data,
+  {"inputCols": ["age", "weight"], "outputCol": "features"}
+)
+return $i
```

Comment:
This test checks that atomic items (currently implemented) can be converted to native Java types to be processed in SparkML.